src/internal/connector/exchange/query_options.go

```diff
@@ -132,16 +132,19 @@ type DeltaRequestBuilderGetRequestConfiguration struct {
 	QueryParameters *DeltaRequestBuilderGetQueryParameters
 }
```

Comment:
nit: this struct and the one above it should be able to be deleted now (can't comment on all of it directly so I can't make a suggestion about it)